test/unit/std/charconv_int_test.cpp

```diff
@@ -231,14 +233,19 @@ TYPED_TEST(integral_from_char_test, hexadicimal_number)
 
 TYPED_TEST(integral_from_char_test, to_chars)
```

Comment:
Can we have a second test with an explicit small number?

This test is great, because is tests this in a generic way, but sometimes it is also valuable to have explicit instances to test.

Like some edge case: `0`, `42`, `1234`

Reply:
I added it, but there is no edge case for all the types that I could put there ..
the implementation's first branch checks if there are 4 (32 bit) or 8 (64 bit) zeros.
So basically every number I add there - and I'm limited by the 8 bit integer - will result in the same code path

Reply:
Thank you :)